src/components/FilterBar.jsx

```diff
@@ -76,14 +60,13 @@ export default function FilterBar(props) {
           items={sortingOptions}
           defaultValue="deadlineSoon"
           removeNone
-          onChange={updateSorting}
+          onChange={udpateSortingParams}
         />
       </Grid>
     </Grid>
   );
 }
 
 FilterBar.propTypes = {
-  changeSortBy: PropTypes.func.isRequired,
-  changeSortFormat: PropTypes.func.isRequired,
+  setSort: PropTypes.func.isRequired,
```

Comment:
So besides a way to update the sorting params, you'll need to pass in the sorting params themselves. Now, this is true for basically every filter param so you can imagine that if there are 10 filters then there we would need properties for at least 10 filter values as well as properties for at least 10 functions for updating them. This doesn't scale well so I suggest instead passing `params` or `queryParams` here and a generic `setParam`/`setQueryParam` function that can be reused by other filter params.